NOTICE.txt

```diff
@@ -249,6 +249,9 @@ LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
 OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION
 WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 
+---
```

Comment:
Can we make sure the license is included here?

Reply:
@jbudz I had added the licence

Reply:
What license is this?  Is there a link to the text we can share?

Reply:
finally I got the link from gainsight https://www.gainsight.com/policy/gainsight-px-license-agreement/ I just added it

Reply:
@jbudz I added the link in `notice.txt`